docs/multi-manager-setup.md

```diff
@@ -11,15 +11,15 @@ weight=3
 
 # High availability in Docker Swarm
```

Comment:
@moxiegirl I think it's a bit tricky for `<tls-config-flags>` in this case because the TLS certs needs to be mounted as a volume to make sure the container sees the certs. I'm afraid that this would cause confusion. Maybe this can be a part added to the TLS docs that explains how to mount a volume with the certs when using the Swarm image (if not done yet). In this case no need to mention it here we can redirect to the TLS docs.

Also we should enforce using `docker run --net=host swarm <...>` because of multiple problems that could occur on the networking side as a good practice.

@vieux @dongluochen Can you confirm? Did I forget something on running Swarm with the Image?